pkgs/by-name/xd/xdg-desktop-portal-termfilechooser/package.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+, meson
+, ninja
+, pkg-config
+, wayland-protocols
+, wayland-scanner
+, inih
+, libdrm
+, mesa
+, scdoc
+, systemd
+, wayland
+}:
+
+stdenv.mkDerivation {
+  pname = "xdg-desktop-portal-termfilechooser";
+  version = "0.1.1";
```

Comment:
Where is this version number from? I'd expect to see the unstable version format used here

Reply:
You are completely correct, this is (like some of other review things) something I mistakenly copied from @nani8ot, who wrote the base for this module and package